src/network/taraxa_capability.cpp

```diff
@@ -1003,6 +1059,11 @@ void TaraxaCapability::onNewTransactions(std::vector<taraxa::bytes> const &trans
 void TaraxaCapability::onNewBlockReceived(DagBlock block, std::vector<Transaction> transactions) {
   LOG(log_nf_dag_prp_) << "Receive DagBlock " << block.getHash() << " #Trx" << transactions.size() << std::endl;
   if (dag_blk_mgr_) {
+    auto status = checkDagBlockValidation(block);
+    if (!status.first) {
+      LOG(log_er_dag_prp_) << "DagBlock validation failed for " << block.getHash().toString();
+      return;
+    }
```

Comment:
[PR 960](https://github.com/Taraxa-project/taraxa-node/pull/960/files) is taking care of this check so this one is going to be redundant